webpack/config.js

```diff
@@ -68,7 +68,11 @@ const config = {
     ]
   },
   resolve: {
-    fallback: { stream: require.resolve("stream-browserify"), zlib: require.resolve("browserify-zlib") },
+    fallback: {
+      stream: require.resolve( "stream-browserify" ),
+      zlib: require.resolve( "browserify-zlib" ),
+      vm: require.resolve( "vm-browserify" )
+    },
```

Comment:
this resolves the error: 
![Screenshot 2024-03-25 at 4 43 34 PM](https://github.com/StateVoicesNational/Spoke/assets/13925271/47fa723e-df4f-4ae6-90e8-55328267b073)